rmw_connext_cpp/src/functions.cpp

```diff
@@ -63,7 +63,6 @@ _create_type_name(
 
 extern "C"
 {
-
```

Comment:
Because cpplint warns about it otherwise.

We could disable the rule `whitespace/blank_line` if we want to. But I am ok with following the style guide as-is.